module1/homework/lengthSort/sort.cpp

```diff
@@ -0,0 +1,19 @@
+#include "sort.hpp"
+
+#include <algorithm>
+
+bool compare(const std::string& s1, const std::string& s2) {
+    return s1.size() < s2.size();
```

Comment:
Will this cover lexicographical comparison?

Reply:
yes